controllers/user-controllers.js

```diff
@@ -0,0 +1,8 @@
+const { fetchUsers } = require("../models/user-models");
+
+exports.getUsers = (req, res, next) => {
+  fetchUsers().then((users) => {
+    console.log(users);
+    res.status(200).send(users);
```

Comment:
Need to send the response as an object with a key of users

Reply:
controller altered